src/main/java/training/supportbank/Helper.java

```diff
@@ -0,0 +1,20 @@
+package training.supportbank;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+//Provides static Helper functions
+public class Helper {
+    public static Set<String> extractNames(List<Transaction> transactions) {
+        Set<String> accountNames = new HashSet<>();
+
+        for (int i = 0; i < transactions.size(); i++) {
+            for(Transaction transaction : transactions) {
```

Comment:
I suspect one of these loops shouldn't be here. The bug's not obvious because it doesn't matter if you add duplicate items to a `Set`. 🙈